app/src/main/java/com/github/epfl/meili/profile/friends/FriendsListRecyclerAdapter.kt

```diff
@@ -25,6 +28,13 @@ class FriendsListRecyclerAdapter : RecyclerView.Adapter<RecyclerView.ViewHolder>
     }
 
     class FriendViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+        init {
+            itemView.setOnClickListener{
+                val intent = Intent(MainApplication.applicationContext(), ChatLogActivity::class.java)
+                MainApplication.applicationContext().startActivity(intent)
+            }
+        }
```

Comment:
What is this for?

Reply:
Sorry this was one way I tried to add transition to the chat activity but forgot to remove it. Thanks for the catch!